rbac/ledger_sync/deltas/handlers.py

```diff
@@ -40,6 +40,7 @@ def update_database(conn, state_change):
     update = get_updater(conn, state_change.block_num)
     remove = get_remover(conn)
     for change in state_change.state_changes:
+        # LOGGER.info("State_change: %s", change)
```

Comment:
Can just take this out if not needed

Reply:
Thanks, another one I forgot to remove.  It's gone now.